Documentation/initial-condition.rst

```diff
@@ -21,12 +21,36 @@ After the simulation is finished the errors between the analytic solution and th
 Use ``cube_c`` to generate the meshes for the convergence tests:
 https://github.com/SeisSol/SeisSol/tree/master/preprocessing/meshing/cube_c
 
-Superimposed Planarwave
------------------------
+Superimposed planar wave
+------------------------
 
 Superimposed three planar waves travelling in different directions. 
 This is especially interesting in the case of directional dependent properties such as for anisotropic materials.
 
+Travelling wave
+---------------
+
+Impose one period of a sinusoidal planar wave as initial condition, see for example the video below.
+There, we impose a P wave travelling to the left. The wave speed at the top and bottom is :math:`2 m/s` and :math:`2.83 m/s` in the middle.
```

Comment:
I would add between the 2 sentences. 
>Note that because of the heterogeneities in the elastic properties, the initially planar wave deviates from planarity.